src/components/extensive/Modal/ModalImageGallery.tsx

```diff
@@ -87,16 +90,41 @@ const ModalImageGallery: FC<ModalImageGalleryProps> = ({
             setSelectedImage={setSelectedImage}
             carouselItem={item => (
               <div className="relative h-full px-24">
-                <div className="absolute left-[calc(50%_-_32px)] bottom-[24px] z-10 flex items-center justify-center rounded-xl bg-darkCustom py-[5px] px-[8px]">
-                  <Text variant="text-13" className="text-white">
-                    {selectedImage + 1} {t("of")} {tabItems.find(tab => tab.id === selectedTab)?.images.length}
+                <div className="absolute left-[24px] top-[24px] z-10 flex translate-x-[75%] flex-col items-center justify-center gap-[2px] rounded-xl border border-white bg-blueCustom-200 p-3">
+                  <Text variant="text-12-bold" className="text-black">
+                    {
+                      tabItems
+                        .find(tab => tab.id === selectedTab)
+                        ?.images.find((_, index: number) => index === selectedImage)?.title
+                    }
+                  </Text>
+                  <Text variant="text-12-light" className="text-black">
+                    {
+                      tabItems
+                        .find(tab => tab.id === selectedTab)
+                        ?.images.find((_, index: number) => index === selectedImage)?.dateCreated
+                    }
+                  </Text>
+                </div>
+                <div className="absolute right-[24px] top-[24px] z-10 flex translate-x-[-75%] items-center justify-center rounded-xl bg-red-100 py-[2px] px-3">
+                  <Text variant="text-12-semibold" className="text-red-200">
+                    {
+                      tabItems
+                        .find(tab => tab.id === selectedTab)
+                        ?.images.find((_, index: number) => index === selectedImage)?.geoTag
+                    }
                   </Text>
                 </div>
                 <ImageWithPlaceholder
                   className="h-full rounded-xl bg-primary-200"
                   alt={t("Image not available")}
                   imageUrl={item.src}
-                ></ImageWithPlaceholder>
+                />
+                <div className="absolute left-[calc(50%_-_32px)] bottom-[24px] z-10 flex items-center justify-center rounded-xl bg-darkCustom py-[5px] px-[8px]">
+                  <Text variant="text-13" className="text-white">
+                    {selectedImage + 1} of {tabItems.find(tab => tab.id === selectedTab)?.images.length}
```

Comment:
Looks like this needs translation. Something like
```
{t('{selected} of {total}', { selected: selectedImage + 1, total: tabItems.find(tab => tab.id === selectedTab)?.images.length })}
```

Reply:
done